Proposed/asc057.md

```diff
@@ -0,0 +1,26 @@
+#### **Title**
+Revision to `Compound`
+
+#### **Authorship**
+Asuka Ota <asuka.ota@strateos.com>
+
+#### **Motivation**
+Some of the challenges in representing molecular structures in 2D include the lack of standardized representation with enough structural data, and the lack of standardized
+method to interpret one representation to another without altering the data it represents (as perceived by the end user). We initially chose to represent Compound using `standard InChI`.
+However, the consensus from the primary users suggest that SMILES is more commonly used method of representing molecule, and each time SMILES is interpreted in InChI (or vice versa) there is a chance of
+misinterpretation due to the challenges mentioned above.
```

Comment:
We should probably be clearer on the sources of misinterpretation. It can be due to a fundamental limitation in the standard InChI with regards to tautomers, etc.
Doesn't have to be in the ASC but an example in the PR or links to issues will be great. E.g. https://github.com/rdkit/rdkit/issues/542

Reply:
Added example. :)